### PR TITLE
Add full model example to the README

### PR DESCRIPTION
**Pull request by ylt**
> https://github.com/openactive/models-ruby#openactive
> - Could we include a fuller example here similar to either https://github.com/openactive/OpenActive.NET#full-models-example or https://github.com/openactive/models-php#models

Also made a tiny additional fix here to pull the @context up to the top (and also improved the ordering of the other @'s)

### CODE REVIEW

File: lib/openactive/helpers/json_ld.rb [hunk 1]
```diff
@@ -33,6 +33,21 @@ def prepare_data_for_serialization(obj, parent = nil, schema: false)
               ]
           end
 
+          # push @'s to the start
+          data = data.sort_by do |k, _v|
+            if k == '@context'
```
Comment (coatezy) [inline lib/openactive/helpers/json_ld.rb]: How about a case statement as an alternative to if/elsif here?

**Review by coatezy — APPROVED** [review]
LGTM. 👍

File: lib/openactive/helpers/json_ld.rb (hunk 1 above)
Comment (ylt) [reply lib/openactive/helpers/json_ld.rb]: Alternatively I was thinking:
```
if k.start_with?('@')
  ["@context", "@type", "@id"].index(k) || 3
else
  4
end
```

or more concisely
```
k.start_with?('@') ? ["@context", "@type", "@id"].index(k) || 3 : 4
```

These are both more obfuscated than a case statement, so will go with that instead.

**Review by thtg88 — APPROVED** [review]

File: lib/openactive/helpers/json_ld.rb (hunk 1 above)
Comment (coatezy) [reply lib/openactive/helpers/json_ld.rb]: Ha - Nice, that would also work. I think the more explicit if/elsif or case statement is clearer to understand though.

**Pull request merged**